server/handlers.go

```diff
@@ -220,27 +672,27 @@ func getTheme(params url.Values, logger *zap.Logger) ring.Theme {
 func handleIdenticon(logger *zap.Logger) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		params := r.URL.Query()
-		pks, ok := params["publicKey"]
-		if !ok || len(pks) == 0 {
+		parsed := ParseParams(logger, params)
+
+		if parsed.PublicKey == "" {
 			logger.Error("no publicKey")
 			return
 		}
-		pk := pks[0]
-		image, err := identicon.Generate(pk)
+
+		image, err := identicon.Generate(parsed.PublicKey)
```

Comment:
`image` variable collides with the `image` package name. Same situation as the `color` package / variable name collision.

Reply:
There is actually a chance that I myself have introduced some of these 😆